libs/scalalib/src/mill/scalalib/SonatypeCentralPublishModule.scala

```diff
@@ -60,16 +62,20 @@ trait SonatypeCentralPublishModule extends PublishModule {
     }
 }
 
-object SonatypeCentralPublishModule extends ExternalModule {
+object SonatypeCentralPublishModule extends ExternalModule with TaskModule {
 
   val defaultCredentials: String = ""
   val defaultReadTimeout: Int = 60000
   val defaultConnectTimeout: Int = 5000
   val defaultAwaitTimeout: Int = 120 * 1000
   val defaultShouldRelease: Boolean = true
 
+  // Set the default command to "publishAll"
+  def defaultCommandName(): String = "publishAll"
+
   def publishAll(
-      publishArtifacts: mill.util.Tasks[PublishModule.PublishData],
+      publishArtifacts: mill.util.Tasks[PublishModule.PublishData] =
+        Tasks.resolveMainDefault("__.publishArtifacts"),
```

Comment:
Avoid potential collisions.

```suggestion
        Tasks.resolveMainDefault("__:PublishModule.publishArtifacts"),
```